app/Platform/Actions/UpdateGamePlayerGames.php

```diff
@@ -39,11 +42,13 @@ public function execute(Game $game): void
                     ->onQueue('player-game-metrics-batch')
                     ->name('player-game-metrics ' . $game->id . ' ' . $page)
                     ->allowFailures()
-                    ->finally(function (Batch $batch) {
+                    ->finally(function (Batch $batch) use ($game) {
                         // mark batch as finished even if jobs failed
                         if (!$batch->finished()) {
                             resolve(BatchRepository::class)->markAsFinished($batch->id);
                         }
+                        // some game metrics depend on the player_games rows
+                        (new UpdateGameMetrics())->execute($game);
```

Comment:
That's a good addition - this was certainly missing when looking at some games' metrics

Reply:
thinking about it this definitely should be dispatching a job instead of running the game metris action directly